homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/pokemons.html

```diff
@@ -0,0 +1,322 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+        <link rel="stylesheet" href="style.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <header class="header">
+            <div class="container header__container">
+                <section class="header__inner">
+                    <img src="image/logo.svg" alt="Logo">
+                    <nav class="navigation">
+                        <ul class="navigation__list">
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="#">Pokemons</a>
+                            </li>
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="index.html">About</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </section>
+            </div>
+        </header>
+        <main class="main">
+            <div class="container">
+                <section class="pokemon">
+                    <h1 class="pokemon__title">Study project "Pokemons"</h1>
+                    <div class="pokemon__cards">
```

Comment:
Don't you think, that these are the sets of the same items? Please add a bit of semantics.
<img width="1349" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/c600050c-d345-4c74-b8a1-eac205fe8fed">

Reply:
`ul` I mean